src/linear_algebra/bilinear_map.lean

```diff
@@ -9,7 +9,9 @@ import linear_algebra.basic
 /-!
 # Basics on bilinear maps
 
-This file provides basics on bilinear maps.
+This file provides basics on bilinear maps. The most general form consider are maps that are
+semilinear in both arguments. They are of type `M →ₛₗ[ρ₁₂] N →ₛₗ[σ₁₂] P`, where `M`, `N` and `P`
+are modules over `R`, `S` and `R₂` respectively, and `ρ₁₂ : R →+* R₂` and `σ₁₂ : S →+* S₂`.
```

Comment:
What about `S₂`? `P` is a module over two different rings, right?

Reply:
Yes -- fixed.